config/custom-environment-variables.yaml

```diff
@@ -112,3 +112,7 @@ ecosystem:
     store: ECOSYSTEM_STORE
     # Badge service (needs to add a status and color)
     badges: ECOSYSTEM_BADGES
+    # List of module names, or objects { name, config } for instantiation to use in sd-setup
+    setup: SD_SETUP_PLUGINS
```

Comment:
Oh I think you're right. But since `SD_SCM_SETUP` is not configured, it will use the value in `default.yaml`. So in this case, nothing will be affected.